.ci/checkDeletedTxt.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+. "$SCRIPT_DIR/helper.sh"
```

Comment:
Is this `.` doing the same as `source "$SCRIPT_DIR/helper.sh"` in the other file? I would rather prefer the latter one for better understanding what is done and being more consistent.